src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/SliderRows/SliderPanel/Header/Header.types.ts

```diff
@@ -0,0 +1,8 @@
+export type HeaderProps = {
+  name: string;
+  label: string;
+  isExpanded: boolean;
+  isMultiple: boolean | undefined;
```

Comment:
maybe you can use just isMultiple?: boolean ?

Reply:
done